include/vsg/vk/Instance.h

```diff
@@ -78,7 +78,18 @@ namespace vsg
             if (pNameFallback) procAddress = reinterpret_cast<T>(vkGetInstanceProcAddr(_instance, pNameFallback));
             return (procAddress);
         }
-
+        // VK_EXT_debug_utils
```

Comment:
Is there a reason why these extensions aren't set up as part of vsg::Extentsions?

https://github.com/vsg-dev/VulkanSceneGraph/blob/master/include/vsg/vk/Extensions.h

Reply:
The only reason is that it's an instance extension. If you don't care about the distinction (and not sure what the difference is on practice), then the pointers could go in vsg: Extensions. I guess that device extensions are particular to one device.

Reply:
I will have think about where best to place things.  It may be appropriate to just create Instance extensions and Device extensions structuress.